### DISCUSSION TITLE
Existe un manual para la carga de un csv en postman?

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#189] [2. Questions and Answers] [answered]
Hola

Tengo que cargar aprox unas 1M a mas de properties y sé que postman se puede. El detalle es que no recuerdo bien los pasos que debía hacer.

Por casualidad hay un manual simple de los pasos para cargar?

Gracias

**Comment by acedward** [comment, marked as answer]
@NicolasPLG la forma más fácil es llamando la api en forma paralela, por la gran cantidad de datos.

Para hacerlo lo más rápido posible debes llamar al endpoint ```POST /api/v2/properties``` con el header de Authorization y con la propiedad en el body. **Debes** estar atento a los 429 porque el rate-limiter te dirá que tan rápido puedes reintentar insertar datos.
 
No estoy seguro si POSTMAN es la mejor forma de insertar 1.000.000 de datos.

**Reply by NicolasPLG** [reply]
@acedward :O entiendo, o qué sería mejor que postman para cargar dichos datos? :(

**Reply by acedward** [reply]
Para 1M de datos, no creo que usar postman sea una buena idea.

**Reply by acedward** [reply]
Creo que manipular tal magnitud de datos, un script sea lo mejor

**Reply by NicolasPLG** [reply]
perfecto, haré un doble primero en la base, ya que encontré unos duplicados entregados por le client y puede que baje bastante.

gracias @acedward

**Reply by NicolasPLG** [reply]
@acedward era eso, el cliente entregó varios elementos duplicados y se redujo bastante la lista 🤣 

Gracias por la ayuda

**Reply by jdurandpaz** [reply]
https://blog.postman.com/looping-through-a-data-file-in-the-postman-collection-runner/
